package.json

```diff
@@ -43,7 +43,9 @@
   "dependencies": {
     "@aws-sdk/client-s3": "^3.26.0",
     "@aws-sdk/lib-storage": "^3.27.0",
-    "@types/eyevinn-iaf": "^1.0.1",
-    "winston": "^3.3.3"
+    "@types/eyevinn-iaf": "^1.0.2"
+  },
+  "optionalDependencies": {
+    "fsevents": "^2.3.2"
```

Comment:
How is this handled when installing it (iaf-plugin-aws-s3) as a package?

Reply:
nvm, I deleted `node_modules` and `package-lock.json` and ran npm install and now the ci tests don't seem to need fsevents